packages/server/tests/acceptance/conformityTests.spec.ts

```diff
@@ -204,6 +208,34 @@ function checkResponseFormat(fileName, actualReponse, expectedResponse) {
   return missingKeys;
 }
 
+function findSchema(file) {
+  const execApisOpenRpcData = require('../../../../openrpc_exec_apis.json');
+  const schema = execApisOpenRpcData.methods.find((method) => method.name === file)?.result?.schema;
+
+  return schema;
+}
+
+function isResponseValid(schema, response) {
+  const validate = ajv.compile(schema);
+
+  const valid = validate(response.result);
+  expect(validate.errors).to.be.null;
+
+  return valid;
+}
+
+function validateBlockNumber() {
+  const schema = {
+    title: 'hex encoded unsigned integer',
+    type: 'string',
+    pattern: '^0x([1-9a-f]+[0-9a-f]*|0)$',
```

Comment:
This regex will fail for:
- hex strings with leading zeros
- the string `"0x"`

Is this the expected behaviour?

Reply:
Sorry, this is leftover from testing, I will remove it